### PR TITLE
INTYGFV-14932: Added PatientAddress to display on signed certificate

### PR DESCRIPTION
**Pull request by UlrikaSopra**

### CODE REVIEW

File: packages/webcert/src/store/certificate/certificateReducer.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import {
```
Comment (SopraErik) [inline packages/webcert/src/store/certificate/certificateReducer.ts:1]: Is the deletion here relevant? Isn't this to do with another feature?

**Review by SopraErik — CHANGES_REQUESTED** [review]
Looks good, but I think the deletion in certificateReducer is probably some mistake?

File: packages/webcert/src/store/certificate/certificateReducer.ts (hunk 1 above)
Comment (UlrikaSopra) [reply packages/webcert/src/store/certificate/certificateReducer.ts:1]: Someone had forgotten to remove it and if I didn't remove it, I got an error in the browser

Comment (SopraErik) [reply packages/webcert/src/store/certificate/certificateReducer.ts:1]: Ah, saw now that it's also in the master branch, and is never used there, so probably from some unmerged branch?
I'll approve the PR

**Review by SopraErik — APPROVED** [review]

**Pull request merged**